docs/integrations/ai-engines/rag.mdx

```diff
@@ -103,102 +99,130 @@ Now you can use the model to answer your questions.
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|-----------------------------------------|
+|   ShopRite Arthritis Pain Acetaminophen is not specifically designed for treating a cold. It may help to temporarily relieve minor aches and pains associated with a cold, but it is not the best product for treating a cold. It is always best to consult with a doctor or pharmacist for the most appropriate medication for treating a cold. | `{"column":["full_ingredients","indications_and_usage","intended_purpose_of_product","active_ingredient"],"sources_content":["ShopRite Arthritis  ..."],"sources_document":["dataframe"],"sources_row":[1,1,1,1]}`   | what product is best for treating a cold? |
+
+
+
+
+### From VectorDB
+
+The following example utilize `rag_engine` to create a model with the `CREATE MODEL` statement and `ChromaDB` database as a knowlege base.
 
-Create the RAG engine providing credentials for LLM you want to use.
 
 ```sql
-CREATE ML_ENGINE rag_writer
+CREATE ML_ENGINE rag_engine
 FROM rag
 USING
-    writer_org_id = "value",
-    writer_api_key = "value";
+    openai_api_key = 'sk-xxx';
 ```
 
-Create a model and embed input data.
+Connect to ChromaDB:
 
 ```sql
- -- Create with writer and embed input data
- CREATE MODEL rag_writer_model
- FROM datasource
-    (SELECT * FROM table)
+CREATE DATABASE chromadb_datasource
+WITH ENGINE = "chromadb",
+PARAMETERS = {
+    "persist_directory": "~/MyProjects"
+}
+```
+
+Create a model using this engine and include the FORM clause:
+
+```sql
+ CREATE MODEL rag_openai_model4
+ FROM mysql_demo_db (select * from demo_fda_context limit 2)
  PREDICT answer
  USING
-    engine="rag_writer",
+    engine="rag_engine",
     top_k=4,
-    llm_type="writer",
+    llm_type="openai",
     summarize_context=true,
-    vector_store_name="faiss",
+    vector_store_name="chromadb",
     run_embeddings=true,
-    vector_store_folder_name='rag_handler_writer_test',
+    vector_store_folder_name='~/MyProjects/rag_handler_openai_test',
     embeddings_model_name="BAAI/bge-base-en",
+    input_column='question',
     prompt_template='Use the following pieces of context to answer the question at the end. If you do not know the answer, just say that you do not know, do not try to make up an answer.
                         Context: {context}
                         Question: {question}
                         Helpful Answer:';
 ```
 
-Now that the model is created, trained, and deployed, you can query for predictions.
+Now you can use the model to answer your questions.
 
 ```sql
- SELECT *
- FROM rag_writer_model
- WHERE question = 'what product is best for treating a cold?';
+SELECT *
+FROM rag_handler_db
+WHERE question='what product is best for treating a cold?';
 ```
+
+On execution, we get:
```

Comment:
General question: in other integrations pages, the images of output are not included. Do you think we should include output images? Then, we should do it for all.

Reply:
Keeping it as discussed